src/scripts/experience/experience.js

```diff
@@ -33,6 +33,16 @@ define([
             this._showStats();
         }
 
+        var colophon = $('.colophon');
```

Comment:
Sounds good, I have noticed this before, just need to get into the habit. Angular also has this kinda convention, though I believe they use a double $ so, I reckon it is safe.